oleesir/store-manager
# #161181977 Users should be able to see the signin page

#### What does this PR do?
implement sign-in-template

#### Description of Task to be completed?
 Implement sign-in-template

#### How should this be manually tested?
* Navigate  to the UI folder and click on signin.html in your browser
* You will see a sign in template

#### What are the relevant pivotal tracker stories?
[#161181977](https://www.pivotaltracker.com/story/show/161181977)

#### Any background context you want to add?
N/A

#### Screenshots
<img width="1280" alt="screen shot 2018-10-12 at 1 24 12 pm" src="https://user-images.githubusercontent.com/17679511/46875423-95ff7180-ce33-11e8-8dc6-c71d3722169b.png">
